src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,331 @@
+    CHAR remoteCandidateId[MAX_CANDIDATE_ID_LENGTH + 1]; //!< Remote candidate that is inspected in RTCIceCandidateStats
+    RTC_STATS_ICE_CANDIDATE_PAIR_STATE state; //!< State of checklist for the local-remote candidate pair
+    BOOL nominated; //!< Flag is TRUE if the agent is a controlling agent and FALSE otherwise. The agent role is based on the
+                    //!< STUN_ATTRIBUTE_TYPE_USE_CANDIDATE flag
+    DefineableUint32 circuitBreakerTriggerCount; //!< Represents number of times circuit breaker is triggered during media transmission
+                                       //!< It is undefined if the user agent does not use this
+    UINT32 packetsDiscardedOnSend; //!< Total number of packets discarded for candidate pair due to socket errors,
+    UINT64 packetsSent; //!< Total number of packets sent on this candidate pair;
+    UINT64 packetsReceived; //!< Total number of packets received on this candidate pair
+    UINT64 bytesSent; //!< Total number of bytes (minus header and padding) sent on this candidate pair
+    UINT64 bytesReceived; //!< Total number of bytes (minus header and padding) received on this candidate pair
+    UINT64 lastPacketSentTimestamp; //!< Represents the timestamp at which the last packet was sent on this particular
+                                    //!< candidate pair, excluding STUN packets.
+    UINT64 lastPacketReceivedTimestamp; //!< Represents the timestamp at which the last packet was sent on this particular
+                                        //!< candidate pair, excluding STUN packets.
+    UINT64 firstRequestTimestamp; //!< Represents the timestamp at which the first STUN request was sent on this particular candidate pair.
+    UINT64 lastRequestTimestamp; //!< Represents the timestamp at which the last STUN request was sent on this particular candidate pair.
+                                 //!< The average interval between two consecutive connectivity checks sent can be calculated:
+                                 //! (lastRequestTimestamp - firstRequestTimestamp) / requestsSent.
+    UINT64 lastResponseTimestamp; //!< Represents the timestamp at which the last STUN response was received on this particular candidate pair.
+    UINT64 totalRoundTripTime; //!< The sum of all round trip time (seconds) since the beginning of the session, based
+                               //!< on STUN connectivity check responses (responsesReceived), including those that reply to requests
+                               //!< that are sent in order to verify consent. The average round trip time can be computed from
+                               //!< totalRoundTripTime by dividing it by responsesReceived.
+    UINT64 currentRoundTripTime; //!< Latest round trip time (seconds)
+    UINT64 availableOutgoingBitrate; //!< Total available bit rate for all the outgoing RTP streams on this candidate pair. Calculated by underlying
```

Comment:
this can be undefined. perhaps we should make them all use definable data types?